### PR TITLE
Documenting partitions <1.9.x> [7678]

### PR DESCRIPTION
**Pull request by IkerLuengo**

### CODE REVIEW

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/153/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/154/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/155/)

File: docs/advanced.rst [hunk 1]
```diff
@@ -58,6 +58,138 @@ This consist of defining a maximum number of data sinks and a maximum size for e
...
+Partitions is a light mechanism to provide data separation among Endpoints:
+
+ * Unlike Domain and Topic, Partitions can be changed dynamically during the life cycle of the Endpoint
+   with little cost. Specifically, no new threads are launched, no new memory is allocated and the
+   change history is not affected. Beware that modifying the Partition membership of endpoints will
+   trigger the announcement of the new QoS configuration, and as a result, new Endpoint matching
+   may occur, depending on the new Partition configuration. Changes on the memory allocation
+   and running threads may occur due to the matching of remote Endpoints.
+
+ * Unlike Domain and Topic, an Endpoint can belong to several Partitions at once. For certain data
```
Comment (MiguelCompany) [inline docs/advanced.rst]:
```suggestion
 * Unlike Domain and Topic, an Endpoint can belong to several Partitions at the same time. For certain data
```

File: docs/advanced.rst [hunk 2]
```diff
@@ -58,6 +58,138 @@ This consist of defining a maximum number of data sinks and a maximum size for e
...
+   to be shared over different Topics, we must use a different Publisher for each Topic,
+   each Publisher sharing its own topic change. A single Publisher can share the same data over different
```
Comment (MiguelCompany) [inline docs/advanced.rst]:
```suggestion
   each Publisher sharing its own history of changes. A single Publisher can share the same data over different
```

File: docs/advanced.rst [hunk 3]
```diff
@@ -58,6 +58,138 @@ This consist of defining a maximum number of data sinks and a maximum size for e
...
+   Partitions, using a single topic change, reducing network overload.
+
+The Partition membership of an Endpoint can be configured on the :class:``qos.m_partitions`` attribute of
+the :class:``PublisherAttributes`` or :class:`SubscriberAttributes`` objects.
```
Comment (MiguelCompany) [inline docs/advanced.rst]:
```suggestion
the :class:``PublisherAttributes`` or :class:``SubscriberAttributes`` objects.
```

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/157/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/158/)

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/159/)

**Pull request merged**